packages/terafoundation_kafka_connector/package.json

```
@@ -21,12 +21,12 @@
     "author": "Terascope, LLC <info@terascope.io>",
     "license": "MIT",
     "dependencies": {
-        "node-rdkafka": "~2.12.0"
+        "node-rdkafka": "~2.13.0"
```

```suggestion
        "node-rdkafka": "~2.14.0"
```

Let's try and revive this MR ... I note that there is a [`node-rdkafka`](https://github.com/Blizzard/node-rdkafka) release `2.14.2` from October that ships [`librdkafka`](https://github.com/confluentinc/librdkafka) release `1.9.2` from August.